redhat-cop/casl-ansible
# combined osp and aws images into one

#### What does this PR do?
This combines the installer-openstack and installer-aws images into one image called casl-ansible. 

#### How should this be manually tested?
Check the instructions in the README. The behavior of either one of the old images should be mimicked depending on the options that you select. 

#### Is there a relevant Issue open for this?
none

#### Other Relevant info, PRs, etc.
none

#### Who would you like to review this?
cc: @redhat-cop/casl
